src/reporter/timex.py

```diff
@@ -70,9 +46,14 @@ def select_time_index_value(headers: dict, notification: dict) -> datetime:
       subscription has to be created with an ``httpCustom`` block as detailed
       in the *Subscriptions* and *Custom Notifications* sections of the NGSI
       spec.
+    - Custom time inde metadata. The most recent custom time index attribute value
+      found in any of the attribute metadata sections in the notification.
     - ``TimeInstant`` attribute.
     - ``TimeInstant`` metadata. The most recent ``TimeInstant`` attribute value
       found in any of the attribute metadata sections in the notification.
+    - ``timestamp`` attribute.
+    - ``timestamp`` metadata. The most recent ``timestamp`` attribute value
+      found in any of the attribute metadata sections in the notification.
```

Comment:
Some pending items:
- We'd need to update the docs in https://github.com/smartsdk/ngsi-timeseries-api/blob/master/docs/manuals/user/index.md
- Target PR towards 0.6.1rc (updating release notes)

Reply:
I moved the target from master to 0.6.1rc and updated the documentation with the selection process. The subscription part should probably also be updated. Right now it only mentions <code>dateCreated></code> and <code>dateModified</code> for the metadata.

By the way, is the <code>dateCreated</code> metadata actually used anywhere in QL?

Reply:
excellent, thanks! good question, IIRC we're not using the `dateCreated` at all, so this might be something worth pruning at some point (not now). Again, this will clearer come with a better understanding of what really happens in Orion with the elements in the 'metadata' section of the subcription (see my other comment)